docs/data-storage.md

```diff
@@ -516,7 +525,7 @@ indices.
 
 The first is an index of non-finalized transactions for an account. This
 contains the next nonce (from the perspective of the last finalized block) for
```

Comment:
Depending on the outcome of the discussion in the PR, this might no longer be the case.